mlem/core/metadata.py

```diff
@@ -43,7 +44,7 @@ def get_object_metadata(
 
 def save(
     obj: Any,
-    path: str,
+    path: Union[str, os.PathLike],
```

Comment:
I see, this covers `mlem.api.load` and `mlem.api.save`. Should we cover all other methods in `mlem.api`?

Reply:
I don't see a pressing need to do those now, because the main purpose was to pass paths for loading and saving. Other APIs have paths for model, which no one really uses the `pathlib` library for.

If needed, we can tackle in another PR.

Reply:
Sorry, didn't get your argument. What is the different between `mlem.api.load` and `mlem.api.apply` in this case? In both places you pass a path. It could be a string and it could be a pathlib object.

Reply:
ok, sure! Then I guess this issue is done, I'll open another one if needed.